conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+
+function errorFunction(int:Signed8 x, int? a) {
+    int:Signed8? _ = x << a; // @error  expected 'int:Signed8?' but found 'int?'
+    int _ = x << a; // @error expected 'int' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional builtin
+             subtypes of integer as the RHS operand.
+Labels: decimal, float, int, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32, int:Unsigned8,
+        shift-expr, optional-type
+
+function errorFunction(int:Signed8 a, int:Signed8? b, int:Unsigned8? c, int:Signed16? d, int:Unsigned16? e,
+                        int:Signed32? f, int:Unsigned32? g) {
+    int:Signed16 _ = a << b; // @error expected 'int:Signed16' but found 'int?'
+    float _ = a << c; // @error expected 'float' but found 'int?'
+    int:Unsigned16 _ = a << d; // @error expected 'int:Unsigned16' but found 'int?'
+    int:Signed16 _ = a << e; // @error expected 'int:Signed16' but found 'int?'
+    decimal _ = a << f; // @error expected 'decimal' but found 'int?'
+    int:Unsigned16 _ = a << g; // @error expected 'int:Unsigned16' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional byte as
+             the RHS operand.
+Labels: byte, int, int:Signed8, optional-type, shift-expr
+
+function errorFunction(int:Signed8 x, byte? a) {
+    int:Signed8 _ = x << a; // @error  expected 'int:Signed8' but found 'int?'
+    int _ = x << a; // @error expected 'int' but found 'int?'
+    byte _ = x << a; // @error expected 'byte' but found 'int?'
+}
+
+Test-Case: output
+Description: Test '<<' expression with 'int:Signed8' operands in function call and method call.
+Labels: function-call-expr, function-defn, int:Signed8, method-call-expr, shift-expr, unary-minus, value:toBalString,
+        value:toString
+
+function init() {
+    int:Signed8 a = -112;
+    int:Signed8 b = -113;
+    int:Signed8 c = 12;
+    int:Signed8 d = 2;
+
+    leftShiftExpr(a << b); // @output -3670016
+    leftShiftExpr(a << d); // @output -448
+    leftShiftExpr(c << d); // @output 48
+    leftShiftExpr(c << b); // @output 393216
+
+    io:println((a << b).toString()); // @output -3670016
+    io:println((a << d).toString()); // @output -448
+    io:println((c << d).toString()); // @output 48
+    io:println((c << b).toString()); // @output 393216
+
+    io:println((a << b).toBalString()); // @output -3670016
+    io:println((a << d).toBalString()); // @output -448
+    io:println((c << d).toBalString()); // @output 48
+    io:println((c << b).toBalString()); // @output 393216
+}
+
+function leftShiftExpr(int x) {
+    io:println(x);
+}
+
+Test-Case: output
+Description: Test '<<' expression with 'int:Signed8' operands as a return statement.
+Labels: DecimalNumber, function-call-expr, function-defn, int, int:Signed8, return-stmt, shift-expr, unary-minus
```

Comment:
Do we need DecimalNumber Label

Reply:
No. It is not necessary here since we have the type descriptor `int:Signed8`. Will remove that label to keep consistency.